### PR TITLE
feat(auth): allow /newsletters access with access token

### PR DESCRIPTION
**Pull request by chenba**
Because:
 - we need to enable Payments to subscribe to newsletters on a user's
   behalf

This commit:
 - add a new scope value for newsletters access
 - add the oauthToken strategy to /newsletters

## Issue that this pull request solves

Closes: #9787

### CODE REVIEW

File: packages/fxa-shared/oauth/constants.js [hunk 1]
```diff
@@ -5,6 +5,7 @@
 module.exports = {
   OAUTH_SCOPE_OLD_SYNC: 'https://identity.mozilla.com/apps/oldsync',
   OAUTH_SCOPE_SESSION_TOKEN: 'https://identity.mozilla.com/tokens/session',
+  OAUTH_SCOPE_NEWSLETTERS: 'https://identity.mozilla.com/account/newsletters',
```
Comment (chenba) [inline packages/fxa-shared/oauth/constants.js:8]: @dannycoates How about it?  Include this in the scopes of the token returned after a passwordless user sign up?

**Review by dannycoates — APPROVED** [review]
works for me!

File: packages/fxa-shared/oauth/constants.js (hunk 1 above)
Comment (dannycoates) [reply packages/fxa-shared/oauth/constants.js:8]: sure

Comment (chenba) [reply packages/fxa-shared/oauth/constants.js:8]: hmm I will change that 'account' to 'accounts' to be consistent with the other scopes.

**Pull request merged**